webrtc.html

```diff
@@ -908,18 +908,18 @@ <h4>
                     <dfn data-idl="">closed</dfn>
                   </td>
                   <td>
-                    The {{RTCPeerConnection}} object's {{RTCPeerConnection/[[IsClosed]]}}
-                    slot is <code>true</code>.
+                    {{RTCPeerConnection/[[IceConnectionState]]}} is
```

Comment:
that internal slot is not defined anywhere yet; or did you mean to reference attribute?

Reply:
never mind it's pending in #2826